beacon_node/beacon_chain/src/data_column_verification.rs

```diff
@@ -473,10 +485,17 @@ fn verify_data_column_sidecar<E: EthSpec>(
     if data_column.kzg_commitments.is_empty() {
         return Err(GossipDataColumnError::UnexpectedDataColumn);
     }
-    if data_column.column.len() != data_column.kzg_commitments.len()
-        || data_column.column.len() != data_column.kzg_proofs.len()
-    {
-        return Err(GossipDataColumnError::InconsistentCommitmentsOrProofLength);
+    let cells_len = data_column.column.len();
+    let commitments_len = data_column.kzg_commitments.len();
+    let proofs_len = data_column.kzg_proofs.len();
+    if cells_len != commitments_len || cells_len != proofs_len {
+        return Err(
+            GossipDataColumnError::InconsistentCommitmentsOrProofLength {
```

Comment:
Why not split this into two errors? Weird to have an "Or" in an error name

Reply:
Alright might as well include it given the size of the PR https://github.com/sigp/lighthouse/pull/7117/commits/5ce829989b132d1acf322e2f88ec2cff7c4bded4